### PR TITLE
Added callback examples

### LINKED ISSUE #1104: Callbacks examples???
Where have they gone?  It seems like they are now missing.  y tho

### PR DESCRIPTION
**Pull request by fehguy**
Fixes #1104 with style, grace, clarity, efficiency and good humor

### CODE REVIEW

File: examples/v3.0/callback-example.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+          required: true
+          description: |
+            the location where data will be sent.  Must be network accessible
+            by the source server
+          schema:
+            type: string
+            format: uri
+            example: https://tonys-server.com
+      responses:
+        201:
```
Comment (MikeRalphson) [inline examples/v3.0/callback-example.yaml]: statusCodes must be quoted in yaml.

**Comment by fehguy** [comment]
FYI hyperlink won't work until merged

File: examples/v3.0/callback-example.yaml (hunk 1 above)
Comment (fehguy) [inline examples/v3.0/callback-example.yaml]: Thanks, done

File: examples/v3.0/callback-example.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+                  subscriptionId:
+                    description: this unique identifier allows management of the subscription
+                    type: string
+                    example: 2531329f-fb09-4ef7-887e-84e648214436
+      callbacks:
+        # the name `onData` is a convenience locator
+        onData:
+          # when data is sent, it will be sent to the `callbackUrl` provided
+          # when making the subscription PLUS the suffix `/data`
+          $request.query.callbackUrl/data:
```
Comment (DavidBiesack) [inline examples/v3.0/callback-example.yaml]: @darrelmiller 's changes (#1103) use `{ }` around expressions - 
```
myWebhook:
  'http://notificationServer.com?transactionId={$request.body#/id}&email={$request.body#/email}}':
```

should callbacks likewise embed expressions in `{}`

**Pull request merged**